### ISSUE TITLE
[Enhancement] Add script icon to userscript header

### ISSUE THREAD

**Issue by ae5960e8-a6fc-491f-b252-898ecf59af95** [DenverCoder1/unedit-for-reddit#79]
The line below can be added to the `// ==UserScript==` area so the logo of the script shows up in our script manages.
[This is the logo I'm talking about.](https://raw.githubusercontent.com/DenverCoder1/Unedit-for-Reddit/3.9.4/images/logo128.png)
```
// @icon https://raw.githubusercontent.com/DenverCoder1/Unedit-for-Reddit/3.9.4/images/logo128.png
```

As you can see, many other scrips take advantage of this feature, and it would make a good addition.
![Screenshot from 2023-01-24 15-38-48](https://user-images.githubusercontent.com/76442625/214338736-2d1e8783-e659-4a0f-989f-c5d8f743b8a8.png)

**Comment by DenverCoder1** [comment]
Sounds great, I didn't know about that.

Maybe the larger logo without the padding could be used:

https://raw.githubusercontent.com/DenverCoder1/unedit-for-reddit/master/images/logo512.png

Do you want to open a PR?

**Comment by ae5960e8-a6fc-491f-b252-898ecf59af95** [comment]
I don't know how the pull request system works.

**Comment by DenverCoder1** [comment]
The steps are basically:
* create a fork of the repo
* create a branch in the fork
* commit changes to the branch
* open a pull request from the GitHub UI with the New Pull Request button

When viewing the script on GitHub, the "edit this file" button (pencil icon) is a shortcut that should do most of the steps automatically including creating a fork and allowing to suggest changes with the commit button at the top.

I can do it pretty quickly, so if you'd prefer I just do it, feel free to let me know.

**Comment by DenverCoder1** [comment]
Closed by #90
